setup.py

```diff
@@ -28,7 +28,7 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-sdk>=1.5.0",
-        "pennylane>=0.13.0",
+        "pennylane @ git+https://github.com/PennyLaneAI/pennylane.git",

```

Comment:
Is this still necessary?

Reply:
Nope! This can be changed to
```suggestion
        "pennylane>=0.15",
```